### ISSUE TITLE
Support deneb hardfork

### ISSUE THREAD

**Issue by chichi13** [ethpandaops/ethereum-metrics-exporter#96]
Since this morning, some metrics are no longer available because of the deneb hardfork:

```bash
Jan 17 12:17:17 devhub-rpc-server-01 ethereum-metrics-exporter[4142094]: {"component":"exporter","error":"failed to parse response: unrecognised data version \"deneb\"","level":"error","module":"consensus/beacon","msg":"Failed to get signed beacon block at finalized","time":"2024-01-17T12:17:17+01:00"}
Jan 17 12:17:17 devhub-rpc-server-01 ethereum-metrics-exporter[4142094]: {"component":"exporter","error":"failed to parse response: unrecognised data version \"deneb\"","level":"error","module":"consensus/beacon","msg":"Failed to get signed beacon block at head","time":"2024-01-17T12:17:17+01:00"}
Jan 17 12:17:17 devhub-rpc-server-01 ethereum-metrics-exporter[4142094]: {"component":"exporter","error":"failed to parse response: unrecognised data version \"deneb\"","level":"error","module":"consensus/beacon","msg":"Failed to get signed beacon block at finalized","time":"2024-01-17T12:17:17+01:00"}
Jan 17 12:17:17 devhub-rpc-server-01 ethereum-metrics-exporter[4142094]: {"component":"exporter","error":"failed to parse response: unrecognised data version \"deneb\"","level":"error","module":"consensus/beacon","msg":"Failed to get signed beacon block at head","time":"2024-01-17T12:17:17+01:00"}
```

Is it planned to support this hardfork ?

**Comment by samcm** [comment]
Thanks for the issue @chichi13! That'll be included in a release asap :)

**Comment by samcm** [comment]
Included in `v0.23.0` https://github.com/ethpandaops/ethereum-metrics-exporter/releases/tag/v0.23.0
